ge09_team04/models/repair.py

```diff
@@ -0,0 +1,38 @@
+from odoo import api, models, fields
+
+
+class Repair(models.Model):
+    _inherit = 'repair.order'
+
+    partner_id = fields.Many2one(compute='_compute_partner_from_registry', store=True)
+    sale_order_id = fields.Many2one(compute='_compute_sale_from_registry')
+    
+    vin = fields.Char(related='motorcycle_registry_entry.vin', readonly=False)
+    mileage = fields.Float(related='motorcycle_registry_entry.current_mileage')
+    motorcycle_registry_entry = fields.Many2one(comodel_name="motorcycle.registry", compute='_find_registry_from_lot_or_vin', store=True)
+
+    @api.depends('motorcycle_registry_entry')
+    def _compute_partner_from_registry(self):
```

Comment:
A computed method should only have the logic for the field assignment, i.e. it does not have to assign other fields or to clear anything else. 

Also if we are only going to assign the value that we have in another table is not it better to use another method for this assignment?